tests/foreman/cli/test_contentview.py

```diff
@@ -209,6 +209,43 @@ def test_negative_create_with_org_name(self):
         with self.assertRaises(CLIReturnCodeError):
             ContentView.create({'organization-id': gen_string('alpha')})
 
+    @tier2
+    @run_only_on('sat')
+    def test_positive_create_with_repo_id(self):
+        """Create content view providing repository id
+
+        @id: bb91affe-f8d4-4724-8b61-41f3cb898fd3
+
+        @assert: Content view is created and repository is associated with CV
+
+        @BZ: 1213097
+        """
+        repo = make_repository({'product-id': self.product['id']})
+        cv = make_content_view({
+            'organization-id': self.org['id'],
```

Comment:
It is needed to specify product when passing repo by name and not needed when passing by id. AFAIK, hammer works in the same way with most dependent entities.
But maybe it would be better to add product 'just in case'